api/predict.py

```diff
@@ -53,24 +52,17 @@ def post(self):
         result = {'status': 'error'}
 
         args = input_parser.parse_args()
-        audio_data = args['audio'].read()
-
-        # clean up from earlier runs
-        if os.path.exists("audio.wav"):
-            os.remove("audio.wav")
 
-        if '.wav' in str(args['audio']):
-            file = open("audio.wav", "wb")
-            file.write(audio_data)
-            file.close()
-        else:
+        if '.wav' not in str(args['audio']):
```

Comment:
[python-magic](https://github.com/ahupp/python-magic) or [mimetypes](https://docs.python.org/3.7/library/mimetypes.html) might be more accurate.

Reply:
Would like to reduce dependencies on external packages. For mimetype I see this [issue](https://stackoverflow.com/questions/44891157/why-wav-format-doesnt-have-same-mimetype-in-different-browsers).